libs/live-nft/src/index.ts

```diff
@@ -130,6 +131,11 @@ export const isNftTransaction = <T>(transaction: T | undefined | null): boolean
   return false;
 };
 
-export const mapChains = (chains: string[], replacements: { [key: string]: string }) => {
-  return chains.map(chain => replacements[chain] || chain);
+export const mapChains = (chains: string[]) => {
+  return chains.map(mapChain);
+};
+
+export const mapChain = (chain?: string) => {
+  if (!chain) return;
+  return replacements[chain] || chain;
```

Comment:
just to understand the functional requirement here, what is the purpose of returning the chain name as a fallback when  not mapped ?

Reply:
because it means that you don't have to replace the name. Solution not perfect but as it's very specific you need to add it manually in the `replacement` record